manifests/charts/dash/templates/prometheus-rules-default.yaml

```diff
@@ -17,7 +17,7 @@ spec:
       rules:
       - alert: ManyNewCVEs
         annotations:
-          description: Fires when the number of new major + critical CVEs in the last 10m exceeds 20
+          description: Fires when the number of new major + critical CVEs in the last 10m exceeds {{ .threshold }}
```

Comment:
Should this have use a .duration like the ones below?

Reply:
The .duration field is actually new